Tasks/AzurePowerShellV3/task.json

```diff
@@ -72,7 +72,7 @@
             "helpMarkDown": "Azure Resource Manager subscription to configure before running PowerShell",
             "visibleRule": "ConnectedServiceNameSelector = ConnectedServiceNameARM",
             "properties": {
-                "EndpointFilterRule": "ScopeLevel == ManagementGroup || ScopeLevel != ManagementGroup"
+                "EndpointFilterRule": "ScopeLevel != AzureMLWorkspace"
```

Comment:
@Ajay-MS Can we have "ScopeLevel == ManagementGroup && ScopeLevel == Subscription" ?

Reply:
No because old endpoints will not have endpoint scope rule and they won't be listed under condition `ScopeLevel == Subscription"`

@romil07 
Can you also update the default filtering rule at Connected Service Component and ARM Service Component Utility.  Otherwise all task will list down AzureMLWorkspace based endpoints too.

Reply:
Service side changes are done.